Assets/Scripts/Pathfinding.cs

```diff
@@ -1,18 +1,32 @@
 using System.Collections;
 using System.Collections.Generic;
 using UnityEngine;
+using Unity.AI;
+using UnityEngine.AI;
 
 public class Pathfinding : MonoBehaviour
 {
+    [SerializeField]
+    private GameObject Player;
+    private Vector3 Target;
+    private NavMeshAgent agent;
+
+
     // Start is called before the first frame update
     void Start()
     {
-        
+        Player = GameObject.FindGameObjectWithTag("Player");
+        agent = gameObject.GetComponent<NavMeshAgent>();
     }
 
     // Update is called once per frame
     void Update()
     {
-        
+        if (Player != null)
+        {
+            Target = Player.transform.position;
```

Comment:
Pathfinding works, but this line just makes the NavMesh Agent move to its own position, so it doesn't move. If we make Target a Transform from which we can extract its Vector3 transform.position, then we can lead the Agent anywhere along the NavMesh that we want